numba/typing/npydecl.py

```diff
@@ -285,6 +285,9 @@ def generic(self, args, kws):
                      "sqrt", "square", "reciprocal",
                      "divide", "mod", "abs", "fabs" ]
 
+if numpy_version >= (1, 15):
```

Comment:
I'm not sure the `math.gcd` impl should be interested in the NumPy version?

Reply:
Isn't this numpy ufunc registration?

Reply:
heh, yes, I read the `mathdecl` and this change as a single file diff :)